### PR TITLE
Support database as a backend for audit logs

### PR DESCRIPTION
**Pull request by akshay196**
### What does this PR change?
PR includes helm chart change to push audit logs to database by default instead of elasticsearch.

### Does the PR depend on any other PRs or Issues? If yes, please list them.

-

### Checklist

I confirm, that I have...

- [ ] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [ ] Updated `CHANGELOG.md`
- [ ] Ran [`helm-docs`](https://github.com/norwoodj/helm-docs) to update docs for chart (if values.yaml file was changed)

### CODE REVIEW

**Comment by akshay196** [comment]
@niravparikh05  There is a sample audit log entries in the database table. Please check if this is as per Paralus backend/frontend expects or not.
[audit_logs_table.txt](https://github.com/paralus/helm-charts/files/10031523/audit_logs_table.txt)

File: charts/ztka/README.md [hunk 1]
```diff
@@ -37,8 +39,9 @@ A Helm chart for Paralus ZTKA.
 | deploy.contour.tls | object | `{}` | TLS properties of the virtual host |
 | deploy.elasticsearch.address | string | `""` | Elasticsearch address. Required when `deploy.elasticsearch.enable` is unset. |
 | deploy.elasticsearch.enable | bool | `false` | Elasticsearch instance is auto deployed and managed by Helm release when true. |
-| deploy.filebeat.enable | bool | `true` | Filebeat is used to collect audit logs into the system. You can disable this if you don't want audit logs. |
+| deploy.filebeat.enable | bool | `false` | Filebeat is used to collect audit logs into the system via elasticsearch. You can disable this if you don't want audit logs or you want to store audit logs into database. |
 | deploy.filebeat.indexPrefix | string | `"ralog"` | You can use this to config the index prefixes for elasticsearch.  This has to match with your filebeat config in `filebeat.daemonset.filebeatConfig.filebeat.yml` |
+| deploy.fluentbit.enable | bool | `true` | of you want to push audit logs to elasticsearch. |
```
Comment (niravparikh05) [inline charts/ztka/README.md:44]:
```suggestion
| deploy.fluentbit.enable | bool | `true` | if you want to push audit logs to database. |
```

**Review by niravparikh05 — COMMENTED** [review]
Need to handle fluentbit restarts, it processes all the auditlogs from start causing duplicate entries.

image

**Comment by akshay196** [comment]
> Need to handle fluentbit restarts, it processes all the auditlogs from start causing duplicate entries.

@niravparikh05 Good catch. Fixed in latest commit.

**Review by niravparikh05 — DISMISSED** [review]
lgtm

**Review by niravparikh05 — APPROVED** [review]
lgtm

**Pull request merged**